vendor/views/vendor_admin.py

```diff
@@ -387,3 +388,48 @@ def get_context_data(self, *args, **kwargs):
         context = super().get_context_data(**kwargs)
         context['title'] = _("Payments with no Order Items")
         return context
+
+
+class PaymentProcessorSiteConfigsListView(ListView):
+    template_name = 'vendor/manage/processor_site_config_list.html'
+    model = SiteConfigModel
+    queryset = SiteConfigModel.objects.all()
```

Comment:
you probably don't want this queryset to be all SiteConfigModels; even though this is the only config on vendor at the moment, if it's integrated with other projects using siteconfigs (such as TC), this will show all unrelated configs as well as vendor settings. i'd suggest filtering by the key

Reply:
This is intended as a helper view for super_users. The view we actually use in TC will be different.

Reply:
becuase of the key, I see what you mean

Reply:
Changed it to:
```

    def get_queryset(self):
        payment_processor = PaymentProcessorSiteConfig()
        return SiteConfigModel.objects.filter(key=payment_processor.key)

```